### PR TITLE
Create YAML for the welcome channel

### PR DESCRIPTION
**Pull request by Oliver-makes-code**

### CODE REVIEW

**Review by intergrav — APPROVED** [review]
Looks good to me

**Review by Ecorous — APPROVED** [review]

**Review by ImperatorStorm — CHANGES_REQUESTED** [review]
Should be named `welcome.yaml` unless I'm missing an inside joke.

File: weclome.yaml [hunk 1]
```diff
@@ -0,0 +1,124 @@
+- type: text
+  text: https://github.com/HelixLauncher/Art/blob/main/branding/banner-launcher/banner-launcher_96h.png?raw=true
+
+- type: embed
+
+  embeds:
+    - title: Welcome to HelixLauncher
+
+      description: >
+        This is the official discord server for the HelixLauncher project
```
Comment (kb-1000) [inline weclome.yaml]:
```suggestion
        This is the official Discord server for the HelixLauncher project.
```

File: weclome.yaml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    Listen to & respect all community team members: >
+      Please take requests from community team members seriously and listen when a team member asks you to do
+      something. If you have a problem with what a team member is asking you to do, we advise that you step away from
+      the situation and send a private message to another team member.
+
+    Keep all projects legal, legitimate & appropriate: >
+      We will not tolerate any projects that may break laws, breach a Terms of Service or an EULA, involve sexual
+      content, or that the moderators determine to be malicious or inappropriate.
+
+    Don’t spam: >
```
Comment (kb-1000) [inline weclome.yaml]:
```suggestion
    Don't spam: >
```

**Comment by Oliver-makes-code** [comment]
> Should be named `welcome.yaml` unless I'm missing an inside joke.

rip

**Review by kb-1000 — APPROVED** [review]

**Review by Astralchroma — APPROVED** [review]

**Review by ImperatorStorm — APPROVED** [review]

File: welcome.yaml [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+
+- type: links
+
+  title: Useful Links
+
+  links:
+    HelixLauncher Website: https://helixlauncher.dev/
+    HelixLauncher on GitHub: https://github.com/HelixLauncher/
+
```
Comment (ENDERZOMBI102) [inline welcome.yaml:119]: maybe add one for the guidelines?

**Review by ENDERZOMBI102 — COMMENTED** [review]
review'd

File: welcome.yaml [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+      level. Try to speak English to the best of your ability, and don't worry if you need to use translation tools. We
+      will not tolerate making fun of people for poor English.
+
+    Listen to & respect all community team members: >
+      Please take requests from community team members seriously and listen when a team member asks you to do
+      something. If you have a problem with what a team member is asking you to do, we advise that you step away from
+      the situation and send a private message to another team member.
+
+    Keep all projects legal, legitimate & appropriate: >
+      We will not tolerate any projects that may break laws, breach a Terms of Service or an EULA, involve sexual
```
Comment (pluiedev) [inline welcome.yaml]: "breach a Term of Service" or "breach our Terms of Service"

**Review by pluiedev — APPROVED** [review]
lgtm except for one thing

File: welcome.yaml (hunk 4 above)
Comment (kb-1000) [reply welcome.yaml]: now that you mention it, the same issue with basing stuff on "may break laws" that was in the guidelines also applies here

**Review by leo60228 — APPROVED** [review]

**Review by ENDERZOMBI102 — APPROVED** [review]

**Review by FractalGeodes — APPROVED** [review]

**Comment by FractalGeodes** [comment]
Only skimmed but looks good

**Review by Ecorous — APPROVED** [review]

**Pull request merged**